addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2092,8 +2093,10 @@ const ListUserValueWidget = UserValueWidget.extend({
     },
     /**
      * @private
+     * @param {boolean} [previewMode=false] if true, the editor will not be
+     *     reloaded.
```

Comment:
Widgets are not supposed to know what's above them in the widget hierarchy: "the editor will not be reloaded" does not mean anything